### ISSUE TITLE
Issue with "HTML attribute reference": missing role attribute

### ISSUE THREAD

**Issue by david-fong** [mdn/content#3614]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes

#### What information was incorrect, unhelpful, or incomplete?

Incomplete: information about the aria [`role`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles) attribute.

Is there a reason that it isn't listed here?

#### What did you expect to see?

A row for the `role` attribute.

#### Did you test this? If so, how?

N/A.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/html/attributes`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/attributes/index.html
* Last commit: https://github.com/mdn/content/commit/1f2db9f0a46b99c254476b5f9445a74b98a6fa30
* Document last modified: 2021-03-27T00:17:29.000Z

</details>

**Comment by sideshowbarker** [comment]
To whoever fixes this: Nowhere in MDN yet do we actually yet have the `role` attribute itself documented — instead we have various roles (_values_ of the `role` attribute) documented — with an index page at https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles. So what needs to be done are the following:

1. Create an https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/role article
2. Add the `role` attribute to https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes, linking to the article in <span>#</span>1.
2. Add the `role` attribute to https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes, linking to the article in <span>#</span>1.

The https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/role article would need to link to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles and maybe to other existing MDN documentation on WAI-ARIA.

**Comment by Khartir** [comment]
I just noticed that several other global attributes are also missing, most notably the `aria-*` and `on*` attributes.
Maybe it would make more sense to just add a link to https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes above the list and remove all global attributes from https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes?
Otherwise global attributes would always have to be added twice.

**Comment by estelle** [comment]
The aria-* attributes are complete.  https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes
Most roles are complete: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles (with the exception of widget roles)

**Comment by estelle** [comment]
Started off by linking to the role page in a11y section until attribute page is created.

**Comment by teoli2003** [comment]
@estelle: I wasn't sure if #14913 was enough to close this issue.

**Comment by estelle** [comment]
Role is not in the HTML specification. The WHATWG spec reads:

> The role attribute is defined in Accessible Rich Internet Applications (ARIA)

I think the solution is to add it as "other" - which I did in https://github.com/mdn/content/pull/16157.

I don't think creating a https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/role article is appropriate or necessary, but maybe I am wrong.
